tensorflow/core/kernels/mlir_generated/gpu_unary_ops_large_tensor_test.cc

```diff
@@ -0,0 +1,48 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ ==============================================================================*/
+
+#include <algorithm>
+#include <cmath>
+#include <complex>
+#include <limits>
+
+#include "tensorflow/core/kernels/mlir_generated/base_ops_test.h"
+#include "tensorflow/core/kernels/mlir_generated/base_unary_ops_test.h"
+
+namespace tensorflow {
+namespace {
+// Test fixture `UnaryOpsTest` that sets the TF device is expected by the TEST
+// macros below.
+class UnaryOpsTest : public UnaryOpsTestBase {
```

Comment:
Is this needed? Can we not reuse the one that is already defined?

Reply:
PTAL

Reply:
Proposing to have a new class for all large tensors. We can add the binary test as well if agreed?